code/datums/hud.dm

```diff
@@ -30,7 +30,10 @@ GLOBAL_LIST_INIT(huds, list(
 	ANTAG_HUD_OBSESSED = new/datum/atom_hud/antag/hidden(),
 	ANTAG_HUD_FUGITIVE = new/datum/atom_hud/antag(),
 	ANTAG_HUD_GANGSTER = new/datum/atom_hud/antag/hidden(),
-	ANTAG_HUD_SPACECOP = new/datum/atom_hud/antag()
+	ANTAG_HUD_SPACECOP = new/datum/atom_hud/antag(),
+	///START VOIDCREW EDITS
+	FACTION_HUD_GENERAL = new/datum/atom_hud/faction()
```

Comment:
this can be too, just requires a little more work, look at `voidcrew/code/vc_world.dm` to see how we do it currently (youd have to create new proc which adds to this list)